Content.Client/Administration/UI/Bwoink/BwoinkControl.xaml.cs

```diff
@@ -74,7 +74,8 @@ public BwoinkControl()
                 if (info.Antag && info.ActiveThisRound)
                     sb.Append(new Rune(0x1F5E1)); // 🗡
 
-                if (info.OverallPlaytime <= TimeSpan.FromMinutes(_cfg.GetCVar(CCVars.NewPlayerThreshold)))
+                var newPlayerThreshold = _cfg.GetCVar(CCVars.NewPlayerThreshold);
```

Comment:
I think this declaration should be moved "higher" in the code branch, because here, it's going to be called a _lot_ and while the actual cost of this is probably not significant, there is just no reason to be constantly reloading the value of this cvar, it's not going to change

Reply:
```cs
var newPlayerThreshold = 0;
_cfg.OnValueChanged(CCVars.NewPlayerThreshold, (show) => { newPlayerThreshold = show; }, true);
```